collector/benchlib/src/benchmark.rs

```diff
@@ -102,19 +107,60 @@ impl BenchmarkGroup {
 }
 
 /// Adds a single benchmark to the benchmark group.
-/// ```ignore
-/// use benchlib::define_benchmark;
 ///
-/// define_benchmark!(group, my_bench, {
-///     || do_something()
-/// });
+/// The first argument should be the `BenchmarkGroup` into which will the benchmark be stored.
```

Comment:
```suggestion
/// `$group` should be the `BenchmarkGroup` into which will the benchmark be stored.
```
And likewise for the other arguments.